jvm-packages/xgboost4j-example/src/main/java/ml/dmlc/xgboost4j/java/example/BasicWalkThrough.java

```diff
@@ -113,7 +115,7 @@ public static void main(String[] args) throws IOException, XGBoostError {
     System.out.println("start build dmatrix from csr sparse data ...");
     //build dmatrix from CSR Sparse Matrix
     DataLoader.CSRSparseData spData =
-        DataLoader.loadSVMFile("../../demo/data/agaricus.txt.train?format=libsvm");
+        DataLoader.loadSVMFile("../../demo/data/agaricus.txt.train");
```

Comment:
For example, this demo was not run in the CI previously; as a result, this line has been broken for a good while. Now it's fixed.

Reply:
What !! this demo was not run before?

Reply:
It was not.